packages/plugin-e2e/src/models/components/PanelEditOptionsGroup.ts

```diff
@@ -0,0 +1,59 @@
+import { Locator } from '@playwright/test';
+import { PluginTestCtx } from '../../types';
+import { ColorPicker } from './ColorPicker';
+import { UnitPicker } from './UnitPicker';
+import { Select } from './Select';
+import { MultiSelect } from './MultiSelect';
+import { Switch } from './Switch';
+import { gte } from 'semver';
+import { RadioGroup } from './RadioGroup';
+
+export class PanelEditOptionsGroup {
+  constructor(private ctx: PluginTestCtx, public readonly element: Locator, private groupLabel: string) {}
+
+  getRadio(label: string): RadioGroup {
```

Comment:
Should be called `RadioGroup` no?

Reply:
I changed it to Radio to align with the plugins API. But I agree with you just changed it to be consistent.